src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -140,8 +140,11 @@ private async Task<(IWebDriver browser, ILogs log)> CreateBrowserAsync(string co
                 opts.AddArgument("--headless");
             }
 
+            opts.AddArgument("--no-sandbox");
```

Comment:
This is fair I think. Do you know of any downsides of doing this?

Reply:
I couldn't find anything that would affect the test. Here's what chromioum had to say about it: https://chromium.googlesource.com/chromium/src/+/master/docs/design/sandbox_faq.md#What-is-the-sandbox